cmd/releaseprocessor/main.go

```diff
@@ -43,6 +45,8 @@ var (
 			"v0.8.0": {}, // v0.8.0 is (intentionally) broken
 		},
 	}
+	// only allow unstable semver releases if they're release candidates in the shape of v1.2.3-rc.4
+	allowedPreReleaseRx = regexp.MustCompile(`^-rc\.\d+$`)
```

Comment:
I believe we don't want to do this because we want the BSR default label to be synonymous with stable, and by [definition](https://semver.org/#spec-item-9) a pre-release in semver is unstable. So the only thing we should improve in the module syncer is the ability to sync protos into non-default labels. Wdyt?

Reply:
> So the only thing we should improve in the module syncer is the ability to sync protos into non-default labels. Wdyt?

Agree, there's an open PR on the backend side that does this ready for review 😄 

Once that one is merged in, this one can sync _any valid semver_, and the BSR backend will take care of pushing to the correct labels accordingly.